### PR TITLE
Feature 3.6/swagger 3.25.1

### PR DESCRIPTION
**Pull request by Simran-B**
Backport of #11530

### CODE REVIEW

File: js/server/assets/swagger/index.html [hunk 1]
```diff
@@ -86,8 +86,8 @@
         ],
         layout: "StandaloneLayout",
         //#region ArangoDB-specific changes
-        // Expand the API documentation by default.
-        docExpansion: "list",
+        validatorUrl: null,
+        docExpansion: "none",
```
Comment (dothebart) [inline js/server/assets/swagger/index.html:90]: hm, I vaguely remember that we didn't want to change this before 3.7 - but I personally don't care.

Comment (Simran-B) [reply js/server/assets/swagger/index.html:90]: Frank and Heiko are okay with the change and I think it's for the better.

Comment (pluma) [reply js/server/assets/swagger/index.html:90]: Can we please keep this as-is as it also affects Foxx? See the other PRs for discussion.

Comment (Simran-B) [reply js/server/assets/swagger/index.html]: Restored to "list".

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/10021/

http://jenkins01.arangodb.biz:8080/view/Documentation/job/arangodb-ANY-examples/629/

**Comment by KVS85** [comment]
Tests blue except known `catch` unrelated failure.

**Pull request merged**